include/gantry/core/constants.hpp

```diff
@@ -0,0 +1,3 @@
+#pragma once
```

Comment:
arent we using python scripts for generating such files with constants now?

Reply:
We are using python scripts to generate constants that are shared between C++ and Python.